python/doc/examples/data_analysis/linear_regression.ipynb

```diff
@@ -75,20 +75,21 @@
      "output_type": "stream",
      "text": [
       "coefficients of the linear regression model =  [0.592409,2.98159]\n",
-      "confidence intervals of the coefficients =  [0.714594, 0.470224]\n",
-      "[3.00321, 2.95996]\n"
+      "confidence intervals of the coefficients =  [0.435545, 0.749274]\n",
+      "[2.95382, 3.00935]\n"
      ]
```

Comment:
why do we get such changes in results here ?

Reply:
Looks like a bug in rot, with level<0.5 even the order of bounds is wrong. If you use level=0.9 instead of 0.1 the ordering gets ok.